pkg/cloudprovider/provider/aws/provider.go

```diff
@@ -95,7 +95,8 @@ var (
 		providerconfig.OperatingSystemCentOS: {
 			description: "CentOS Linux 7 x86_64 HVM EBS*",
 			// The AWS marketplace ID from AWS
-			owner: "679593333241",
+			owner:       "679593333241",
```

Comment:
is it necessary to use the id instead of "aws-marketplace"? the latter would be better readable.

Reply:
Well, for CoreOS and Ubuntu the owner ids are actually something else than the marketplace. We could probably use both for readability? I am not sure if the name is unique

Reply:
I just want to make sure the owner is immediately understood. and "aws-marketplace" is less arbitrary id.

Reply:
Oh well - there is the comment above. So my comment is just about cosmetics.